docs/docs/how-to/cloud/adding-a-custom-domain.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Adding a Custom Domain to your site on Gatsby Cloud
+description: Learn how to add a custom domain to your site on Gatsby Cloud
+---
+
+Before you can add a custom domain, you'll need to be hosting with Gatsby Cloud. If you haven't already, [turn on Gatsby Hosting](https://support.gatsbyjs.com/hc/en-us/articles/360061641114).
+
+ 
+
+## 1. Add a Domain
+If you just enabled Gatsby Hosting you should already be in the right place. Otherwise, go to **Site Settings > Hosting**. Now, click **Add Domain** above the domain list to start the process
+
+![Click Add Domain](../../images/mceclip8.png) 
+
+In the **Please enter a domain** field, add one of the following:
+
+- your apex or root domain (this is your custom domain without the www. part, e.g. mysite.com)
+- your subdomain (e.g., www.mysite.com or store.mysite.com)
+![Enter your domain](../../images/hosting-domain-modal.png)
+
+Whichever one you choose will become the **primary domain** for your site.
+
+ 
+
+### Redirecting to the Primary Domain
```

Comment:
In this and other documents the headings are formatted incorrectly: https://www.gatsbyjs.com/contributing/gatsby-style-guide/#format-titles-and-headers

Reply:
Corrected to ensure proper content nesting.